examples/using-redux/gatsby-browser.js

```diff
@@ -4,7 +4,7 @@ import { Provider } from 'react-redux'
 
 import createStore from './src/state/createStore'
 
-exports.replaceRouterComponent = ({ history }) => {
+export const replaceRouterComponent = ({ history }) => {
```

Comment:
Could you make this change to `gatsby-ssr.js` also? Currently `gatsby build` fails.

Reply:
This file use es6 import, so I use es6 export.
I fixed `gatsby build`.